core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.java

```diff
@@ -111,17 +115,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
       String body = getBody();
 
+
       if (allowLogsCheckbox.isChecked()) {
-        File appDirectory = new File(Environment.getExternalStorageDirectory() + "/Kiwix");
-        File logFile = new File(appDirectory, "logcat.txt");
-        if (logFile.exists()) {
-          Uri path =
-            FileProvider.getUriForFile(this,
-              getApplicationContext().getPackageName() + ".fileprovider",
-              logFile);
-          emailIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
-          emailIntent.putExtra(Intent.EXTRA_STREAM, path);
-        }
+        File file = fileLogger.writeLogFile(this);
```

Comment:
have you checked the contents of this log file? this command runs a `logcat -c`
![image](https://user-images.githubusercontent.com/3358282/86892938-d8351000-c0f8-11ea-85ce-d9a927bbf65c.png)
https://developer.android.com/studio/command-line/logcat

The previous code just located the file that was created.